exporter/tanzuobservabilityexporter/metrics_test.go

```diff
@@ -353,23 +489,23 @@ func TestSumConsumerErrorOnSend(t *testing.T) {
 	deltaMetric := newMetric(
 		"test.delta.metric", pmetric.MetricTypeSum)
 	sum := deltaMetric.Sum()
-	mi := metricInfo{Metric: deltaMetric, Source: "test_source", SourceKey: "host.name"}
+	mi := metricInfo{Metric: deltaMetric, Source: "test_source", SourceKey: "host.name", ResourceAttrs: map[string]string{"res_attr_key": "res_attr_value"}}
```

Comment:
This test is supposed to test what happens if there is an error sending a Sum metric to wavefront.  It might be best to leave ResourceAttrs unset in this test so that it defaults to nil and have a separate dedicated test called something like "TestSumConsumerResourceAttrs" that tests that the resource attrs and existing tags are properly merged.  Consider doing the same thing with the other types of consumers as well.

Reply:
done